project1/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Cormorant+Garamond:400,600|Lexend+Mega&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+
+
+
+
+
+    <div class="wrapper">
+    <h1>The Crane Wife</h1>
+    <h2><a href="https://www.theparisreview.org/blog/author/cjhauser/">By CJ Hauser</a> &ensp;  July 16, 2019</h2>
+
+    <blockquote>“I was going to do the squirrel but then I thought, that just isn’t CJ. <i>This</i> is CJ.”</blockquote>
```

Comment:
Per the discussion in class, the `i` element should be used for:
- technical terms
- foreign language phrases
- fictional character's thoughts

What you want here instead is `em`, which places emphasis on the text.
```suggestion
    <blockquote>“I was going to do the squirrel but then I thought, that just isn’t CJ. <em>This</em> is CJ.”</blockquote>
```